lib/system.nim

```diff
@@ -2763,9 +2763,9 @@ when defined(nimvarargstyped):
     ## pretends to be free of side effects, so that it can be used for debugging
     ## routines marked as `noSideEffect <manual.html#pragmas-nosideeffect-pragma>`_.
 else:
-  proc echo*(x: varargs[expr, `$`]) {.magic: "Echo", tags: [WriteIOEffect],
```

Comment:
You should not change this. But since ``nimvarargstyped`` is always defined these days, you can remove this hack for backwards compatibility.